generator/internal/descriptor_utils.cc

```diff
@@ -632,11 +632,35 @@ bool CheckParameterCommentSubstitutions() {
   return all_substitutions_used;
 }
 
+/// If a service name mapping exists, return the new name.
+/// Parses a command line argument in the form:
+/// {"service_name_mappings": "service_a:new_service_a,service:new_service"}.
+std::string GetServiceNameMappingOrReturnDefault(VarsDictionary const& vars,
```

Comment:
nit: `GetEffectiveServiceName` seems to capture the intent and be less Java-like.

Reply:
Done